ynput/ayon-houdini
# Use `rop.opengl` family instead of `review`

## Changelog Description
Use `rop.opengl` family instead of `review` in review plugins that expect only a review using opengl nodes.
Also, remove unnecessary logic that is used to skip instances that are not using opengl.


## Additional review information
This PR is used as a show case to test https://github.com/ynput/ayon-core/pull/994
without it the attributes applied to `rop.opengl` won't be visible in the publisher. in this case it'd be the validate  

| with develop branch | with the core PR |
|--|--|
| ![image](https://github.com/user-attachments/assets/5e45a721-9376-4232-a0de-0721fb49ba06) | ![image](https://github.com/user-attachments/assets/4cd18a7a-78e5-4844-9597-701df2c56fbd) |

## Testing notes:
1. Launch Houdini via AYON.
2. Create a render (with review toggle enabled) and review instances.
3. Publish, (You shouldn't notice any difference, other than opengl plugins only work with `Review (OpenGL)` products.)
